python/lsst/ts/tunablelaser/component.py

```diff
@@ -2,75 +2,77 @@
 
 """
 import logging
-import pty
-import os
+from .ascii import TCPIPClient
 from . import hardware
-from .ascii import SerialCommander
-from .mock_server import MockSerial
 
 
 class LaserComponent:
     """The class that implements the TunableLaser component.
 
     Parameters
     ----------
-    port: `str`
+    port : `str`
         The name of the USB port that the laser connection is located.
-    configuration: `dict`
+    configuration : `dict`
         A dict that is created from :func:`laser_configuration`
-    simulation_mode: `bool`
+    simulation_mode : `bool`
```

Comment:
The "port" and "configuration" constructor arguments are gone. Please delete the doc strings (or move them, if those arguments have moved to other methods that are not yet documented).